README.md

```diff
@@ -113,3 +113,11 @@ The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## 🛡️ License
+
+LinkFree is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
```

Comment:
I'm sorry but this doesn't comply with our contribution guidelines. Seems to be a copy paste from a different project, additionally Passager is not released under MIT License but under A-GPL V3.